app/webpacker/components/RegistrationsV2/Registrations/RegistrationList.jsx

```diff
@@ -95,7 +96,15 @@ export default function RegistrationList({ competitionInfo }) {
   const changeSortColumn = (name) => dispatch({ type: 'CHANGE_SORT', sortColumn: name });
 
   const [psychSheetEvent, setPsychSheetEvent] = useState();
-  const [psychSheetSortBy, setPsychSheetSortBy] = useState('single');
+  const [psychSheetSortBy, setPsychSheetSortBy] = useState('average');
+
+  useEffect(() => {
+    if (psychSheetEvent === '333mbf') {
+      setPsychSheetSortBy('single');
+    } else {
+      setPsychSheetSortBy('average');
+    }
+  }, [psychSheetEvent]);
```

Comment:
This is not a good usage of `useEffect`. In essence, you're only recomputing the initial value for `useMemo` above. So, you should replace this whole effect hook with a single line along the lines of `const default sort = isMultiBld ? 'single' : 'average'` and then pass that new value as initializer: `useMemo(psychSheetEvent)`

Reply:
Done!